docker-nlp-toolset/start-nlp-toolset.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -e
+
+NLP_TOOLSET_DIR=/NlpToolset
+INIT_SCRIPT_PATH=$NLP_TOOLSET_DIR/init.sh
+
+if [ -f $INIT_SCRIPT_PATH ];
+then
+	echo "Running custom init script: $INIT_SCRIPT_PATH"
+	chmod +x $INIT_SCRIPT_PATH
+	$INIT_SCRIPT_PATH
+else
+	echo "No init script found at $INIT_SCRIPT_PATH, you can see an example of the recommended script at https://github.com/Maluuba/NlpToolset/blob/master/init.sh"
+fi
+
+source /etc/profile.d/rvm.sh
+
+cd $NLP_TOOLSET_DIR
+
+cp script/nlptoolset.conf /etc/init/nlptoolset.conf
+
+rvm use jruby-1.7.10@nlp-toolset
+
+bundle install
+
+rake db:migrate
+
+chown -R tomcat7:tomcat7 /usr/share/tomcat7/nlptoolset/db
+
+export HOME=/usr/share/tomcat7
+
+if [ "${RAILS_ENV}" == 'production' ]; then
+	mvn package -Dnlp.toolset.rails.env=$RAILS_ENV
+	service tomcat7 stop
+	rm -rf /var/lib/tomcat7/webapps/ROOT*
+	cp $NLP_TOOLSET_DIR/target/NlpToolset.war /var/lib/tomcat7/webapps/ROOT.war
+	service tomcat7 restart
+
+	start nlptoolset
+else
+	cp -R $HOME/.ssh/ /root/
```

Comment:
I'll need to run this on a server with a public IP to test.  I should just be able to test on an ec2 instance, right?